internal/api/flows.go

```diff
@@ -32,7 +32,6 @@ type FlowCreate struct {
 
 // FlowUpdate is a subset of Flow used when updating flows.
 type FlowUpdate struct {
-	Name *string  `json:"name"`
```

Comment:
There is no `name` field in the [Flow update API](https://app.prefect.cloud/api/docs#tag/Flows/operation/update_flow_api_accounts__account_id__workspaces__workspace_id__flows__id__patch).